### ISSUE TITLE
Exporting WMO Sets are Missing Items when Exported Multiple Times

### ISSUE THREAD

**Issue by briochie** [briochie/wow.unity#1]
All testing was done with the "Always Overwrite Files" attribute set to true for my configs.

When exporting WMOs, either by themselves or along with ADTs, they do not export all of their M2 objects in the selected set when the WMO and doodad set have been exported before in the same session. Restarting the wow.export client fixes this.

It happens consistently with `Set_$DefaultGlobal`, but I haven't been able to replicate it with a different set selected as well.

##Example:
WMO: `world/wmo/kalimdor/buildings/dwarf/ol_dwarven_guardtower.wmo[112763]`:
Export path: `D:\Programming\Game Projects\Test Exports`
WMO Sets: `Set_$DefaultGlobal`, `Set_B`

**Fresh Session:**
1. Export WMO to folder
2. All objects export
3. Delete all files and sub-directories
4. Return to App, export without changing settings
5. All objects export
6. Disable `Set_B`
7. Export
8. Some M2 objects missing

The ModelPlacement File:
[ol_dwarven_guardtower_ModelPlacementInformation.csv](https://github.com/briochie/wow.unity/files/10889495/ol_dwarven_guardtower_ModelPlacementInformation.csv)

**Object Missing:**
- `..\..\..\..\generic\dwarf\passivedoodads\lanterns\new_small_lantern01.obj`

###Log:
[runtime.log](https://github.com/briochie/wow.unity/files/10889492/runtime.log)

**Comment by briochie** [comment]
Exporting ADTs multiple times, I get a lot more warnings (999+ in the unity log) from my package, reporting the obj file doesn't exist.

**Comment by oyvind-stromsvik** [comment]
Right. This was one of the things I mentioned on Discord that you said you were aware of. I didn't see this issue.

I've also noticed that I have to reimport various buildings for them to get all their doodads. In the initial import many of them get zero doodads, but then I can just select all the files (.obj, .csv etc.) for the respective building in the project browser, right click and reimport, and this causes the prefab to be correctly populated with all the expected tables, chairs etc.

Why it failed initially I don't know, but I was trying to import 28 tiles at once, including Stormwind, and Stormwind fails 100% of the time for me, and seemingly causes other things to fail as well. I tried leaving it for hours to see, but nothing happened. It just hangs on importing stormwind_set0.obj or something like that.

I've also noticed that multiple exports from wow.export are sometimes needed to get everything. I've exclusively been exporting tiles, with every checkbox ticked off in the exporter. Sometimes wow.export adds additional files on subsequent exports.

These may be 3-4 different issues for all I know, but they didn't really feel concrete enough to report separately, at least not for the time being :p
